internal/batch/workflow.go

```diff
@@ -0,0 +1,62 @@
+package batch
+
+import (
+	"context"
+	"os"
+	"time"
+
+	wferrors "github.com/artefactual-labs/enduro/internal/workflow/errors"
+
+	"go.uber.org/cadence/workflow"
+)
+
+const (
+	BatchWorkflowName              = "batch-workflow"
+	BatchWorkflowID                = "batch-workflow"
+	BatchWorkflowStateQueryHandler = "batch-state"
```

Comment:
This is unused and safe for removal. A complex workflow could use a query handler to communicate its state but the state in `batch-workflow` is pretty simple, I don't think we will need one for now.